machine_test.go

```diff
@@ -138,7 +135,22 @@ func TestMicroVMExecution(t *testing.T) {
 		exitchannel <- m.Wait(vmmCtx)
 		close(exitchannel)
 	}()
-	time.Sleep(2 * time.Second)
+
+	vmmAlive := false
+	const maxRetry = 5
+	for i := 0; i < maxRetry; i++ {
```

Comment:
Is this worth generalizing since it's used in a couple of places as something like`func waitVmmAlive(client, maxRetries) error`?

Then in the tests anywhere you want to wait for VMM to startup you can do:
```go
if err := waitVmmAlive(m.client, maxRetries); err != nil {
        t.Fatal(err)
}
```

Reply:
Done